src/libraries/Common/src/System/Security/Cryptography/Asn1/OaepParamsAsn.xml.cs

```diff
@@ -11,11 +11,11 @@ namespace System.Security.Cryptography.Asn1
     [StructLayout(LayoutKind.Sequential)]
     internal partial struct OaepParamsAsn
     {
-        private static ReadOnlySpan<byte> DefaultHashFunc => new byte[] { 0x30, 0x09, 0x06, 0x05, 0x2B, 0x0E, 0x03, 0x02, 0x1A, 0x05, 0x00 };
+        private static ReadOnlySpan<byte> DefaultHashFunc => [0x30, 0x09, 0x06, 0x05, 0x2B, 0x0E, 0x03, 0x02, 0x1A, 0x05, 0x00];
```

Comment:
The `.xml.cs` files are generated. These changes will get undone the next time the generator runs.

We should fix the generator so the changes are not undone the next time it runs. The line you are looking for is:

https://github.com/dotnet/runtime/blob/53eaf7d0077e9f815acc970b852fbe79feab4782/src/libraries/Common/src/System/Security/Cryptography/Asn1/asn.xslt#L256

It should be changed to

```plain
private static ReadOnlySpan&lt;byte&gt; <xsl:call-template name="DefaultValueField"/> =&gt; [<xsl:value-of select="@defaultDerInit"/>];
```

Reply:
The "ASN regenerated files" build errors are occurring because the generator is not matching what is checked in. Once the generator and source agree the build errors should go away.

Reply:
Thanks.